packages/material/scss/input/_variables.scss

```diff
@@ -22,9 +39,9 @@ $kendo-input-font-size-md: $font-size-lg !default;
 $kendo-input-line-height-md: 1.25 !default;
 
 $kendo-input-padding-x-lg: map-get( $spacing, 4 ) !default;
-$kendo-input-padding-y-lg: map-get( $spacing, 2 ) + map-get( $spacing, thin ) !default;
+$kendo-input-padding-y-lg: map-get( $spacing, 2 ) !default;
 $kendo-input-font-size-lg: $font-size-lg !default;
-$kendo-input-line-height-lg: 1.25 !default;
+$kendo-input-line-height-lg: 1.5 !default;
```

Comment:
we can use [$line-height-lg](https://github.com/telerik/kendo-themes/blob/develop/packages/material/scss/typography/_variables.scss#L30) here as the values are matching.